include/tcpdf_static.php

```diff
@@ -128,39 +128,6 @@ public static function getTCPDFProducer() {
 		return "\x54\x43\x50\x44\x46\x20".self::getTCPDFVersion()."\x20\x28\x68\x74\x74\x70\x3a\x2f\x2f\x77\x77\x77\x2e\x74\x63\x70\x64\x66\x2e\x6f\x72\x67\x29";
 	}
 
-	/**
```

Comment:
Why removing these functions?
It will break compatibility with older PHP versions.

Reply:
magic quotes support is dropped after php 5.3, thus this code is not needed as only php 5.4+ is supported

see https://www.php.net/manual/en/info.configuration.php#ini.magic-quotes-runtime

Reply:
Can you demonstrate that this lib is not compatible with older versions than the supported one?